frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/backgroundables/OverrideForkPointBackgroundable.java

```diff
@@ -55,6 +55,7 @@ private void overrideForkPoint(IManagedBranchSnapshot branch, ICommitOfManagedBr
       setOverrideForkPointConfigValues(myProject, root, branch.getName(), forkPoint, branch.getPointedCommit());
     }
 
+    // required since the change of .git/config is not considered as a change to VCS (and detected by the listener)
```

Comment:
let's put some comments on the calls - probably it could be enforced by some script (?)